src/main/java/com/hazelcast/web/HazelcastHttpSession.java

```diff
@@ -340,12 +349,33 @@ public boolean isStickySession() {
         return stickySession;
     }
 
+    public boolean isNeedNotify() {
+        return needNotify;
+    }
+
+    public void setNeedNotify(boolean needNotify) {
+        this.needNotify = needNotify;
+    }
+
     public void updateReloadFlag() {
         for (Map.Entry<String, LocalCacheEntry> entry : localCache.entrySet()) {
             if (!entry.getValue().isDirty()) {
                 entry.getValue().setReload(true);
             }
         }
+    }
 
+    /**
+     *  To prevent the eviction of an active session from the distributed map,
+     *  reset the idle time for this session on cluster.
+     */
+    public void notifyCluster() {
+        try {
+            webFilter.getClusteredSessionService().containsSession(id);
```

Comment:
It will be better if we can use an async call here, such as `getAsync`. Otherwise, it will make a sync call to the remote cluster on each get call, which will cause performance drop.

Reply:
Done